### PR TITLE
Release 11.1.0

### PR DESCRIPTION
**Pull request by wyardley**
#### This Pull Request (PR) fixes the following issues
- Add support for oom_score_adj (#877)
- CLI Environment Fixes (#876)
- make sure the rabbitmq_version method actually returns the version (#874)
- Use mocked facts in tests (#873)
- Enable Puppet 7 support (#881)
- Add auto cluster configuration support (#883)
- Compatibility with camptocamp/systemd 3.x (#886)
- puppetlabs/apt: Allow 8.x (#884)
- puppet/archive: Allow 5.x (#884)
- puppetlabs/stdlib: allow 7.x (#884)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v11.0.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v11.0.0) (2021-01-16)
+## [Unreleased](https://github.com/voxpupuli/puppet-rabbitmq/tree/HEAD)
```
Comment (bastelfreak) [inline CHANGELOG.md:7]: this looks odd, it should be 11.1.0 and not unreleased. did you generate the changelog before updating metadata.jdon?

Comment (wyardley) [reply CHANGELOG.md:7]: Derp. You’re right. I’ll regenerate

**Comment by bastelfreak** [comment]
let's include https://github.com/voxpupuli/puppet-rabbitmq/pull/880 here

**Comment by bastelfreak** [comment]
we can also include https://github.com/voxpupuli/puppet-rabbitmq/pull/881 if you like

**Comment by wyardley** [comment]
Fixed changelog and added #881

Are all the remaining failures in #880 expected?

**Comment by wyardley** [comment]
@fatpat: you want to take a look at the generated docs in my branch?

**Comment by wyardley** [comment]
@bastelfreak: my suggestion is that we ship this as-is and release the other changes after the failing CI checks are resolved, either as part of the planned breaking release, or before it.

**Comment by fatpat** [comment]
> @fatpat: you want to take a look at the generated docs in my branch?

all good to me

**Review by bastelfreak — APPROVED** [review]
I'm not sure  ifthe changelog is still 100% correct, it might needs to be rebased before the release

**Comment by wyardley** [comment]
I checked just now - seems like the branch is up to date, changelog is up to date except for the date, and the generated docs are correct as well.

Went ahead and pushed up an update with the correct date.

**Review by bastelfreak — APPROVED** [review]
thanks! feel free to merge & run travis_release

**Comment by wyardley** [comment]
Eh, went ahead and added #886 - why not

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**